Jetchat/app/src/main/java/com/example/compose/jetchat/conversation/Conversation.kt

```diff
@@ -186,14 +190,15 @@ fun Messages(
     scrollState: ScrollState,
     modifier: Modifier = Modifier
 ) {
+
+    val scope = rememberCoroutineScope()
     Box(modifier = modifier) {
 
-        ScrollableColumn(
-            scrollState = scrollState,
-            reverseScrollDirection = true,
+        Column(
```

Comment:
Still not switching to `LazyColumn`? I understand why given that we only have `LazyListState.smoothScrollBy` and no `smoothScrollTo`, but this also means that a real-world chat app with Jetpack Compose is pretty unrealistic at the moment.

Reply:
https://github.com/android/compose-samples/issues/371